posthog/templates/email/invite.html

```diff
@@ -1,18 +1,34 @@
 {% extends "email/base_transactional.html" %} {% load posthog_assets %} {% block section %}
-<p>
+
+<p class="text-center">
     <strong>{{ invite.created_by.first_name }}</strong> ({{ invite.created_by.email }}) has invited you to join
-    {{invite.organization.name }} on PostHog.<br /><br />
-    PostHog is a product analytics tool that will help you better understand your users.<br />
+    <strong>{{invite.organization.name }}</strong> on PostHog. PostHog is a product analytics tool that will help you
+    better understand your users.<br />
     Build the best product <i>together</i>.
 </p>
+{% if invite.message %}
+<div class="invite-message">
+    <div class="invite-message-divider"></div>
+    {{ invite.message }}
+</div>
+{% endif %}
 <div class="mb mt text-center">
-    <a class="button" href="{{ invite }}">Join {{ invite.organization.name }} now</a>
+    <a class="button primary" href="{{ invite }}">Join now</a>
```

Comment:
Is `{{ invite }}` right here? Would that give us the invite object instead of the link?